mas-foundation/src/main/java/com/ca/mas/core/io/http/TrustedCertificateConfigurationTrustManager.java

```diff
@@ -162,4 +165,22 @@ public void checkServerTrusted(X509Certificate[] chain, String s) throws Certifi
     public X509Certificate[] getAcceptedIssuers() {
         return new X509Certificate[0];
     }
+
+    @Override
+    public boolean verify(String hostname, SSLSession session) {
+        if (config.isHostnameVerifierEnabled()) {
+            if (session.getPeerHost() != null && config.getHost().getHost().equals(session.getPeerHost())) {
+                return true;
+            } else {
+                try {
+                    throw new SSLPeerUnverifiedException("Cannot verify hostname: " + hostname);
```

Comment:
Throwing the exception and catching it immediately doesn't make any sense.
You can remove this try-catch block.